File: system-test/dotnet/SystemTests/Latest/ExternalSystemContractTests/Clock/BaseClockContractTest.cs
using SystemTests.Latest.ExternalSystemContractTests.Base;

namespace SystemTests.Latest.ExternalSystemContractTests.Clock;

public abstract class BaseClockContractTest : BaseExternalSystemContractTest
{
    [Fact]
    public async Task ShouldBeAbleToGetTime()
    {
        (await Scenario()
            .Given().Clock().WithTime()
            .Then().Clock())
            .HasTime();
    }
}

File: system-test/java/src/test/java/com/optivem/eshop/systemtest/latest/contract/clock/BaseClockContractTest.java
package com.optivem.eshop.systemtest.latest.contract.clock;

import com.optivem.eshop.systemtest.latest.contract.base.BaseExternalSystemContractTest;
import org.junit.jupiter.api.Test;

public abstract class BaseClockContractTest extends BaseExternalSystemContractTest {
    @Test
    void shouldBeAbleToGetTime() {
        scenario
                .given().clock().withTime()
                .then().clock().hasTime();
    }
}
